web/src/app/services/loi/loi.service.ts

```diff
@@ -79,6 +79,14 @@ export class LocationOfInterestService {
     return this.lois$;
   }
 
+  getJobLocationsOfInterest$(
+    jobId: string
+  ): Observable<List<LocationOfInterest>> {
+    return this.lois$.pipe(
```

Comment:
I see you actually added it here; you can just call it from the component?

Nit: -> getLoisByJobId$()